ios/Podfile

```diff
@@ -7,12 +7,9 @@ require_relative '../node_modules/@react-native-community/cli-platform-ios/nativ
 require_relative '../node_modules/react-native-unimodules/cocoapods.rb'
 
 target 'ZulipMobile' do
-  # Pods from React Native
-  use_react_native!
+  config = use_native_modules!
 
-  # "Autolinking": automatically link pods that depend on React
-  # Native, unless omitted in our own `react-native.config.js`.
-  use_native_modules!
+  use_react_native!(:path => config[:reactNativePath])
```

Comment:
This isn't super well explained, but it looks like their claim for
    this being needed starts by mentioning
    @react-native-community/cli@e949e234b, which was released in v4.10.1
    of @react-native-community/cli.
    
    And `yarn why` tells me that we're on v4.14.0 of that package, so it

This probably means the intermediate state with `config["reactNativePath"]` is broken, right? So probably we should just squash these two changes together.